onnxruntime/test/contrib_ops/element_wise_ops_test.cc

```diff
@@ -245,13 +246,15 @@ TEST(MathOpTest, ComplexMulConj) {
   tester.AddOutput<float>("C", {4, 2}, output_data);
 
   std::vector<std::unique_ptr<IExecutionProvider>> execution_providers;
+#ifdef USE_CUDA
   execution_providers.push_back(DefaultCudaExecutionProvider());
+#elif USE_ROCM
```

Comment:
should be consistent here, either `#if #elif` or `#if defined   #elif defined`

Reply:
Fixed in 82078eed0abc808a34f6c6aaad91ce2a3f7cf6f1.